cloudbeesjenkins-dockerdatacenter/README.md

```diff
@@ -7,34 +7,37 @@
 </a>
 https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/cloudbeesjenkins-dockerdatacenter
 ## Azure Partner Quickstarts Overview
-***Azure Partner Quickstart Templates*** provide customers with a highly automated process to launch enterprise ready first and 3rd party ISV solution stacks on Azure in a pre-production environment. The **Azure Partner Quickstart** effort is complimentary to the [Azure Marketplace test drive program](https://azure.microsoft.com/en-us/marketplace/test-drives/). These fully baked stacks enable customers to quickly stand up a PoC or Piloting environments and also integrate it with their systems and customization.
+***Azure Partner Quickstart Templates*** provide customers with a highly automated process to launch enterprise ready 1st and 3rd party ISV solution stacks on Azure in a pre-production environment. The **Azure Partner Quickstart** effort is complimentary to the [Azure Marketplace test drive program](https://azure.microsoft.com/en-us/blog/azure-marketplace-test-drive/). These fully baked stacks enable customers to quickly stand up a Proof of Concept or Pilot environments, as well as integrate it with their existing systems and configurations.
 
-Customers benefit greatly from solution templates because of the ease with which they can stand up enterprise-grade, fully integrated stacks on Azure. The extensive automation and testing of these solutions will allow them to spin up pre-production environments with minimal manual steps and customization.  Most importantly, customers now have the confidence to transition the solution into a fully production-ready environment with confidence.
+Customers benefit greatly from solution templates because of the ease with which they can stand up enterprise-grade, fully integrated stacks on Azure. The extensive automation and testing of these solutions will allow them to spin up pre-production environments with minimal manual steps and customization.  Most importantly, customers can have the confidence to transition the solution into a fully production-ready environment once ready.
```

Comment:
once ready...  Once the solution/stack is ready or once the customer is ready?  maybe reword...